apps/meteor/tests/end-to-end/apps/video-conferences.ts

```diff
@@ -577,7 +577,8 @@ describe('Apps - Video Conferences', () => {
 							expect(res.body.room)
 								.to.have.a.property('fname')
 								.that.is.a('string')
-								.that.satisfies((msg: string) => msg.startsWith('Chat History'));
+								.that.satisfies((msg: string) => !msg.startsWith('Chat History'))
+								.that.satisfies((msg: string) => msg.includes('Chat History'));
```

Comment:
Hey, if we could have a test to cover the case, when we explicitly provide the place for [date]. wdys ?